hokusai-app-updater/README.md

```diff
@@ -66,3 +47,31 @@ user@artsy:~$ gh auth logout
 ? Are you sure you want to log out of github.com account 'artsyuser'? Yes
 ✓ Logged out of github.com account 'artsyuser'
 ```
+
+## Usage
+
+* Prepare a change script (can be bash, etc.). See [examples](examples/).
+
+* Run `update_apps.sh` to apply changes and push PRs.
+
+```
+$ pwd
+/home/user/code/opstools/hokusai-app-updater
+user@artsy:~/code/opstools/hokusai-app-updater$ ./update_apps.sh
+    Usage: ./update_apps.sh path_to_change_script(relative to this dir) path_to_project_list path_to_source_code_root_dir branch_name commit_message(also pr title/body) pr_reviewer(also assignee)
+```
+
+- `~/tmp/list`: path to file that lists the projects you wish to change. The list is likely different for different tasks.
+```
+user@artsy:~/tmp$ cat list
+convection
+pulse
+```
+
+`/home/user/code`: root dir for source code. Each project's source code dir must be under this root dir. For example:
+```
+user@artsy:~/code/convection$ pwd
+/home/user/code/convection
+```
+
+When making changes to `convection` for example, the script will go into `/home/user/code/convection`. Therefore, project names in the list must match dir names in source root.
```

Comment:
I just put a `Setup` section ahead of the `Usage section` and included `gh` setup there.